src/react/projects/spark-core-react/src/SprkPromo/SprkPromo.js

```diff
@@ -0,0 +1,233 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { SprkLink, SprkButton } from '@sparkdesignsystem/spark-core-react';
+
+const SprkPromo = (props) => {
+  const { 
+    title,
+    subtitle,
+    additionalClasses,
+    additionalClassesContent,
+    cta,
+    ctaText,
+    ctaHref,
+    ctaAnalytics,
+    ctaIdString,
+    imgSrc,
+    imgAlt,
+    imgLinkHref,
+    imgLinkAnalytics,
+    imgLinkIdString,
+    additionalClassesImgLink,
+    idString,
+    isFlag,
+    mediaRev,
+    hasBorder,
+    children,
+    ...other
+  } = props;
+  
+  return (
+    <div
+      className={
+        classnames(
+          'sprk-c-Promo',
+          'sprk-o-Stack',
+          {'sprk-c-Promo--flag': isFlag},
+          {'sprk-c-Promo--bordered': hasBorder},
+          additionalClasses
+        )}
+      data-id={idString}
+      {...other} >
+
+      {imgSrc && isFlag && !mediaRev && 
+        <SprkLink 
+          variant="plain"
+          href={imgLinkHref}
+          additionalClasses={
+            classnames(
+              'sprk-o-Stack__item--fourth@s',
+              'sprk-o-Stack__item',
+              additionalClassesImgLink
+            )}
+          analyticsString={imgLinkAnalytics}
+          idString={imgLinkIdString} >
+
+          <img
+              className="sprk-c-Promo__image--flag"
+              alt={imgAlt}
+              src={imgSrc} />
+        </SprkLink>
+      }
+
+      {imgSrc && !mediaRev && !isFlag && 
+        <SprkLink
+          variant="plain"
+          href={imgLinkHref}
+          additionalClasses={
+            classnames(
+              'sprk-o-Stack__item',
+              'sprk-o-Stack__item--half@s',
+              additionalClassesImgLink
+            )}
+          analyticsString={imgLinkAnalytics}
+          idString={imgLinkIdString} >
+
+          <img
+            className="sprk-c-Promo__image"
+            alt={imgAlt}
+            src={imgSrc} />
+        </SprkLink>
+      }
+
+      <div className={
+        classnames(
```

Comment:
we should have a convo about what our standard is, but I've typically seen the dynamic classes separated and declared above